pkg/imagefilter/formatter_test.go

```diff
@@ -92,16 +91,31 @@ func TestResultsFormatter(t *testing.T) {
 			"@WARNING - the output format is not stable yet and may change\ntest-distro-1:\n  test_type: [ test_arch, test_arch2 ]\n",
 		},
 	} {
-		res := make([]imagefilter.Result, len(tc.fakeResults))
+		res := make([]Result, len(tc.fakeResults))
 		for i, resultSpec := range tc.fakeResults {
 			res[i] = newFakeResult(t, resultSpec)
 		}
 
 		var buf bytes.Buffer
-		fmter, err := imagefilter.NewResultsFormatter(imagefilter.OutputFormat(tc.formatter))
+		fmter, err := NewResultsFormatter(OutputFormat(tc.formatter))
 		require.NoError(t, err)
 		err = fmter.Output(&buf, res)
 		assert.NoError(t, err)
 		assert.Equal(t, tc.expectsOutput, buf.String(), tc)
 	}
 }
+
+func TestSupportedOutputFormats(t *testing.T) {
```

Comment:
I think we should test the sorted-ness here as well as it seems like an important feature of the `SupportedOutputFormats` function :)